openslides_backend/services/authentication.py

```diff
@@ -50,9 +50,8 @@ def get_user(self, headers: Headers) -> int:
             try:
                 user_id = body["user_id"]
             except (TypeError, KeyError):
-                raise AuthenticationException(
-                    "Empty or bad response from authentication service."
-                )
+                # TODO: remove this and throw an AuthentificationException when permissions are implemented.
```

Comment:
Please revert this change because this is done in #73.

Reply:
Yes, rebased on master